src/modules/ToDoList.js

```diff
@@ -0,0 +1,55 @@
+// import Task from './modules/Task.js';
+export default class ToDoList {
+  constructor(taskList = []) {
+    this.taskList = taskList;
+  }
+
+  get getTaskList() {
+    return this.taskList;
+  }
+
+  set setTaskList(newTaskList = []) {
+    if (!newTaskList) return;
+    this.taskList = newTaskList;
+  }
+
+  getLength() {
+    return this.taskList.length;
+  }
+
+  addTask(task) {
+    this.taskList.push(task);
+  }
+
+  removeTask(index) {
+    this.taskList.splice(index, 1);
+  }
+
```

Comment:
- [ ] Good job :+1: :100: implementing the functionality for removing a task. But I noticed that it is not working when clicking the **Trash** button. Make sure to fix that to match the project requirements.

- [ ] Also once you have fixed the above issue you should make sure that deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

Reply:
- [ ] As requested by the previous code reviewer I highly recommend to fix the indexes after deleting an item because after deleting the second task I can't see the indexes are updated 🔥🚀✨💯💯 so kindly work on it to fix the problem


![image](https://user-images.githubusercontent.com/58553711/208212870-eb7a8a8b-98b8-4a6a-b43c-82f09d6b653c.png)